### PR TITLE
CHANGELOG and Installation Instructions

### PR DESCRIPTION
**Pull request by mprimeaux**
Signed-off-by: Michael Primeaux <michael.primeaux@mac.com>

### Description
- Amended CHANGELOG for both `opensearch` and `opensearch-dashboards` charts to reflect version bump to 1.0.3.
- Amended status badges to GitHub-generated markdown.
- Amended installation instructions in `opensearch` and `opensearch-dashboards` charts.
- Added README for `opensearch-dashboards`
- Restrict linting and testing to only when chart-related files change so we are not required to increment the chart version for unrelated commits.
 
### Issues Resolved
This PR, in part, resolves bug #55. 
 
### Check List
- [✔️] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by mprimeaux** [comment]
@DandyDeveloper @smlx @TheAlgo I made a second commit that restricts triggering of the linting and testing workflow to Chart changes. This allows the community to make changes to documentation and other non-chart related files without forcing a version increment of the `opensearch` and `opensearch-dashboards` charts.

**Comment by mprimeaux** [comment]
@peterzhuamazon  It appears the removal of the README in the charts directory is also part of [PR #57](https://github.com/opensearch-project/helm-charts/pull/57).

**Comment by peterzhuamazon** [comment]
> @peterzhuamazon It appears the removal of the README in the charts directory is also part of [PR #57](https://github.com/opensearch-project/helm-charts/pull/57).

That PR is a bit stale as well (no pun intended), I am actually ok to merge it if @TheAlgo resolve the conflicts.

**Comment by mprimeaux** [comment]
@DandyDeveloper @smlx @TheAlgo My latest commit relaxed the trigger of linting and testing based on modification of the GitHub Actions workflow file itself. If we do not relax this then the implication is that any changes to this file would force a version bump of the chart, which, IMHO, is not compliant with the SemVer 2 as applied to the charts.

**Comment by TheAlgo** [comment]
> @DandyDeveloper @smlx @TheAlgo My latest commit relaxed the trigger of linting and testing based on modification of the GitHub Actions workflow file itself. If we do not relax this then the implication is that any changes to this file would force a version bump of the chart, which IMHO, is not compliant with the SemVer 2 as applied to the charts.

Yes I agree we need to draw a line on the version bump.

**Comment by TheAlgo** [comment]
> > @peterzhuamazon It appears the removal of the README in the charts directory is also part of [PR #57](https://github.com/opensearch-project/helm-charts/pull/57).
> 
> That PR is a bit stale as well (no pun intended), I am actually ok to merge it if @TheAlgo resolve the conflicts. All the questions we asked there are enhancements.

Apologies for delay. I have resolved the conflicts anyways. Thanks

File: .github/workflows/lint-test.yaml [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 name: Lint and Test Charts
 
-on: [pull_request, workflow_dispatch]
+on: 
+  pull_request:
+    paths:
+      - 'charts/**/ci/**'
+      - 'charts/**/templates/**'
+      - 'charts/**/Chart.yaml'
+      - 'charts/**/values.yaml'
```
Comment (peterzhuamazon) [inline .github/workflows/lint-test.yaml:10]: Why change to only these paths?

File: charts/opensearch-dashboards/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Helm >= 2.17.0
+* We recommend you to have 8 GiB of memory available for this deployment, or at least 4 GiB for the minimum requirement. Else, the deployment is expected to fail.
+
+## Installing
+
+Once you've added this Helm repository as per the repository-level [README](../../README.md#installing)
+then you can install the chart as follows:
+
+```shell
+helm install my-release opensearch/opensearch-dashboardss
```
Comment (peterzhuamazon) [inline charts/opensearch-dashboards/README.md:21]: `opensearch-dashboardss`

**Comment by mprimeaux** [comment]
This PR has been succeeded by PR #81 .

**Pull request closed without merging**